common/common/srr.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "group.h"
+#include <pack/pack.h>
+
+
+namespace fty::common::srr {
+
+static constexpr const char* Channel       = "FTY.Q.GROUP.SRR";
+static constexpr const char* FeatureName   = "automatic-groups";
+static constexpr const char* ActiveVersion = "1.0";
+
+struct SrrGroupObj : public pack::Node
+{
+    pack::ObjectList<Group> m_groupList = FIELD("groupList");
```

Comment:
when you use pack struct you always use a field as getter/setter member.
srrgroup.groupList....
so, m_ prefix is somehow for private/protected members